core/trino-main/src/main/java/io/trino/server/PluginManager.java

```diff
@@ -275,4 +282,9 @@ public static PluginClassLoader createClassLoader(String pluginName, List<URL> u
             PluginClassLoader create(String pluginName, List<URL> urls);
         }
     }
+
+    public interface PluginInstaller
```

Comment:
extract this class to separate file. It is not implementation detail of PluginManager, it is more an API.